src/chart/generateCategoricalChart.tsx

```diff
@@ -255,13 +255,25 @@ const getTooltipContent = (
 
 /**
  * Returns tooltip data based on a mouse position (as a parameter or in state)
+ * @param  {String} chartName chart name
  * @param  {Object} state     current state
  * @param  {Array}  chartData the data defined in chart
  * @param  {String} layout     The layout type of chart
  * @param  {Object} rangeObj  { x, y } coordinates
  * @return {Object}           Tooltip data data
  */
-const getTooltipData = (state: CategoricalChartState, chartData: any[], layout: LayoutType, rangeObj?: any) => {
+const getTooltipData = (
+  chartName: string,
+  state: CategoricalChartState,
+  chartData: any[],
+  layout: LayoutType,
+  rangeObj?: any,
+): object => {
+  // PieChart is temporarily returning null because the 'calculateActiveTickIndex' is returning an incorrect activeIndex
```

Comment:
One concern with this modification is that it no longer returns payload and index in the mouse handler of PieChart. While this fixes a part that often returns incorrect data and causes bugs, it could be a breaking change for users

Reply:
Thats certainly a breaking change that will be noticed, I don't think we'll be able to fix it for now if thats the only solution